assessment_framework_eng.md

```diff
@@ -0,0 +1,982 @@
+# Responsible and Trustworthy Data Science - Evaluation framework
+
+The [evaluation framework](#evaluation-framework-to-assess-the-maturity-of-an-organisation) below is the result of the participatory work initiated in the spring of 2019 by Substra Foundation and ongoing since then. It is based on the identification of the risks that we are trying to prevent by aiming for a responsible and trustworthy practice of data science, and best practices to mitigate them. It also brings together for each topic technical resources that can be good entry points for interested organisations.
+
+## Evaluation framework to assess the maturity of an organisation
+
+The evaluation is composed of the following 6 sections:
+
+- [Section 1 - Protecting personal or confidential data](#section-1---protecting-personal-or-confidential-data)
+- [Section 2 - Preventing bias, developing non-discriminatory models](#section-2---preventing-bias-developing-non-discriminatory-models)
+- [Section 3 - Assessing model performance rigorously](#section-3---assessing-model-performance-rigorously)
+- [Section 4 - Ensuring model reproducibility and establishing the chain of accountability](#section-4---ensuring-model-reproducibility-and-establishing-the-chain-of-accountability)
+- [Section 5 - Using models responsibly and in confidence](#section-5---using-models-responsibly-and-in-confidence)
+- [Section 6 - Anticipating, monitoring and minimising the negative externalities of data science activities](#section-6---anticipating-monitoring-and-minimising-the-negative-externalities-of-data-science-activities)
+
+---
+
+### Section 1 - Protecting personal or confidential data
+
```

Comment:
Il faut qu'in ajoute les keywords (je ne comprenais pas pourquoi les regex ne catchaient pas ^^)

```suggestion
**[Data privacy]**  
```
ou data protection

Reply:
peut être plutot data protection, "data privacy" ça fait un peu trop "differential privacy" non ?

Reply:
Non, c'est bien